src/main/java/com/exadel/sober/services/PromiseService.java

```diff
@@ -0,0 +1,51 @@
+package com.exadel.sober.services;
+
+import com.exadel.sober.models.Promise;
+import com.exadel.sober.models.Reason;
+import com.exadel.sober.repositories.PromiseRepository;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.util.List;
+@Service
+public class PromiseService {
+    private PromiseRepository promiseRepository;
+    private ReasonService reasonService;
+
+
+    public PromiseService(PromiseRepository promiseRepository, ReasonService reasonService) {
+        this.reasonService = reasonService;
+        this.promiseRepository = promiseRepository;
+    }
+
+    public List<Promise> getPromisesForClient(Integer userId) {
+        List<Promise> promises = promiseRepository.findAllByUserId(userId);
```

Comment:
make sure that findAllByUserId returns an empty list if user does not have promises, because if not - you will get nullPointerException in the next step. Besides getPromisesForClient must always return empty list is no promises because it us normal behavior if user doesn't not have promises

Reply:
Please make some feedback to my comments, even if we discuss it in stanUp meeting or if you checked and it's works fine. Because for now it looks like you did nothing.